RELEASE-NOTES.md

```diff
@@ -122,7 +232,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 -
 
 
-[Unreleased]: https://lc.llnl.gov/bitbucket/projects/ATK/repos/axom/compare/commits?targetBranch=refs%2Ftags%2Fv0.2.9&sourceBranch=refs%2Fheads%2Fdevelop&targetRepoId=1066
-[Version 0.2.9]: https://lc.llnl.gov/bitbucket/projects/ATK/repos/axom/compare/commits?targetBranch=refs%2Ftags%2Fv0.2.8&sourceBranch=refs%2Ftags%2Fv0.2.9&targetRepoId=1066
+[Unreleased]:    https://github.com/LLNL/axom/compare/v0.3.0...develop
```

Comment:
Thanks @gzagaris.

Could you please add/update these links as well?
```
 [Unreleased]:   https://github.com/LLNL/axom/compare/v0.3.1...develop 
 [Version 0.3.1]: https://github.com/LLNL/axom/compare/v0.3.0...v0.3.1
 [Version 0.3.0]: https://github.com/LLNL/axom/compare/v0.2.9...v0.3.0 
```

Reply:
Fixed. Thanks.